test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -828,6 +829,132 @@ def init_proc_mon(self):
                                         frequency=freq)
                 self._process_monitoring = True
 
+    def revert_pbsconf(self):
+        """
+        Revert the following about the pbs.conf file:
+        - start bits
+        - PBS_SERVER
+        - PBS_SCP
+        - PBS_CORE_LIMIT
+        """
+        primary_server = self.servers.values()[0]
+
+        # Revert server hosts' pbs.conf
+        for server in self.servers.values():
+            cmds_to_exec = []
+            dmns_to_start = 0
+            pbs_conf_filepath = self.du.get_pbs_conf_file(server.hostname)
+            pbs_conf_val = self.du.parse_pbs_config(server.hostname,
+                                                    pbs_conf_filepath)
+            if pbs_conf_val is None or len(pbs_conf_val) < 1:
+                raise ValueError("Could not parse pbs.conf on host %s" %
+                                 (server.hostname))
+
+            # Set all start bits
+            if (pbs_conf_val["PBS_START_SERVER"] != "1"):
+                pbs_conf_val["PBS_START_SERVER"] = "1"
+                dmns_to_start += 1
+                cmds_to_exec.append(["server", "start"])
+            if (pbs_conf_val["PBS_START_SCHED"] != "1"):
+                pbs_conf_val["PBS_START_SCHED"] = "1"
+                cmds_to_exec.append(["sched", "start"])
+                dmns_to_start += 1
+            if self.moms is None or len(self.moms) == 0:
+                if (pbs_conf_val["PBS_START_MOM"] != "1"):
+                    pbs_conf_val["PBS_START_MOM"] = "1"
+                    cmds_to_exec.append(["mom", "start"])
+                    dmns_to_start += 1
+            else:
+                pbs_conf_val["PBS_START_MOM"] = "0"
+            if self.comms is None or len(self.comms) == 0:
+                if (pbs_conf_val["PBS_START_COMM"] != "1"):
+                    pbs_conf_val["PBS_START_COMM"] = "1"
+                    cmds_to_exec.append(["comm", "start"])
+                    dmns_to_start += 1
+            else:
+                pbs_conf_val["PBS_START_MOM"] = "0"
```

Comment:
why are we turning mom off when we are able to find comm?

Reply:
Thanks for catching that